xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/less/drawer.less

```diff
@@ -20,87 +20,127 @@
 .drawer--right .drawer-nav {
   right: -@xwiki-drawer-width;
 }
 
-.drawer-nav {
+
+
+#tmDrawer.drawer-nav {
   background-color: @xwiki-drawer-bg;
   width: @xwiki-drawer-width;
-  z-index: 5300;
+  right: 0;
+  overflow: hidden;
+  height: 100vh;
+  margin-right: -@xwiki-drawer-width;
+  transition: margin-right .6s cubic-bezier(0.190, 1.000, 0.220, 1.000);
+  padding: 0;
+  border: 0;
+  // We overwrite max-height over browser defaults for dialog
+  max-height: 100vh;
+  
+  &::backdrop {
+    opacity: 0;
+    background-color: #fff - @xwiki-page-content-bg;
+    transition: opacity .6s cubic-bezier(0.190, 1.000, 0.220, 1.000);
+  }
+
+  &[open]:not(.drawer-transitioning) {
+    margin-right: 0;
+    &::backdrop {
+      opacity: 0.2;
+    }
+  }
 
-  // Brand =================================================================
-  .drawer-brand {
-    background-color: @xwiki-drawer-brand-bg;
-    border-bottom: 1px solid @xwiki-drawer-brand-border;
-    font-weight: 400;
-    line-height: inherit;
-    padding: 20px 16px;
+  & .drawer-menu {
+    position: relative;
+    margin: 0;
+    padding: 0;
+    list-style: none;
 
-    a {
+    & .drawer-menu-item {
+      font-size: 1rem;
       display: block;
-      color: @xwiki-drawer-brand-link-color;
-      padding: 0;
-
-      &:hover, &:focus {
-        color: @xwiki-drawer-brand-link-hover-color;
-        text-decoration: none;
-      }
+      padding: .75rem;
+      text-decoration: none;
+      color: @xwiki-drawer-menu-item-color;
     }
 
-    // Avatar styling
-    img {
-      border-radius: 50%;
-      float: left;
-      height: 50px;
-      width: 50px;
+    & .drawer-menu-item:hover {
+      color: @xwiki-drawer-menu-item-hover-color;
+      background-color: transparent;
     }
 
-    .brand-links {
-      float: left;
-      margin-left: 16px;
-      margin-top: 8px;
+    // Brand =================================================================
+    .drawer-brand {
+      display: flex;
+      justify-content: space-between;
+      align-items: center;
+      gap: 4px;
+      background-color: @xwiki-drawer-brand-bg;
+      border-bottom: 1px solid @xwiki-drawer-brand-border;
+      font-weight: 400;
+      line-height: inherit;
+      padding: 20px 16px;
 
       a {
-        font-size: 14px;
+        display: block;
+        color: @xwiki-drawer-brand-link-color;
+        padding: 0;
+
+        &:hover, &:focus {
+          color: @xwiki-drawer-brand-link-hover-color;
+          text-decoration: none;
+        }
       }
 
-      // Resetting avatar styling
+      // Avatar styling
       img {
-        border-radius: 0;
-        float: none;
-        height: auto;
-        width: auto;
+        border-radius: 50%;
+        height: 50px;
+        width: 50px;
       }
 
-      .brand-user {
-        color: @xwiki-drawer-brand-user-color;
-        font-weight: 400;
+      .brand-links {
+        flex-grow: 1;
+        margin-left: 16px;
+
+        a {
+          font-size: 14px;
```

Comment:
I see that a fixed font size was used before, but shouldn't this actually be relative to the base font size to also respect, e.g., accessibility settings for larger fonts?

Reply:
:+1:  Addressed in https://github.com/xwiki/xwiki-platform/pull/2282/commits/9064120d8d06bf71d7f2921d3707cbd74399af23

Reply:
Note that accessibility settings for larger fonts as of now are in a CSS file, so using `@font-size-base` wouldn't fix it.

I think this is a change I'll make on the new accessibility settings